apps/arena/priv/config.json

```diff
@@ -79,7 +79,7 @@
     "end_game_interval_ms": 1000,
     "shutdown_game_wait_ms": 10000,
     "natural_healing_interval_ms": 300,
-    "zone_shrink_start_ms": 35000,
+    "zone_shrink_start_ms": 999999999,
```

Comment:
Change this back

Reply:
Whoops, this happens more than i would like haha. restored!